src/appkit.cs

```diff
@@ -4037,6 +4156,14 @@ interface NSColorList : NSSecureCoding {
 		bool WriteToUrl ([NullAllowed] NSUrl url, [NullAllowed] out NSError error);
 	}
 
+	[Protocol]
```

Comment:
not availability information ?

Reply:
Negative. The header (NSColorPanel.h) had nothing, nor did our tooling. They added a number of protocols without it, and at least a few appear to be backwards added somehow maybe, I can't tell and don't like guessing.

Reply:
10.14 according to docs https://developer.apple.com/documentation/appkit/nscolorchanging?language=objc, I think we should add it